sdk/docs/README.md

```diff
@@ -0,0 +1,29 @@
+# Introduction
+The Turing SDK is a Python tool for interacting with the Turing API, and complements the existing Turing UI available 
+for managing router creation, deployment, versioning, etc.
+
+It not only allows you to build your routers in an incremental and configurable manner, it 
+also gives you the opportunity to write imperative scripts to automate various router modification and deployment 
+processes, hence simplifying your workflow when interacting with Turing API.
+
+## What is the Turing SDK?
+The Turing SDK is entirely written in Python and acts as a wrapper, around the classes automatically generated (by 
+[OpenAPI Generator](https://github.com/OpenAPITools/openapi-generator)) from the OpenAPI specs written for the Turing 
+API. These generated classes in turn act as an intermediary between raw JSON objects that are passed in HTTP 
+requests/responses made to/received from the Turing API.
+
+![Turing SDK Classes](./assets/turing-sdk-classes.png)
```

Comment:
I think this can be the raw URL (https://github.com/gojek/turing/blob/main/sdk/docs/assets/turing-sdk-classes.png?raw=true) so that once this PR is merged to master, Pypi can load the image from the raw URL. Otherwise, the image can't be loaded with the relative path.